lib/ui/pages/pin_entry.dart

```diff
@@ -0,0 +1,121 @@
+import 'dart:async';
+
+import 'package:get/get.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:pin_code_fields/pin_code_fields.dart';
+import 'package:pispapp/controllers/ephemeral/pin_entry_controller.dart';
+import 'package:pispapp/ui/widgets/moja_button.dart';
+
+class PinEntry extends StatelessWidget {
+
+  // Handles animation for error
+  // Must be created within the widget since a new stream should be created to avoid
+  // "Stream has already been listened to" error
+  StreamController<ErrorAnimationType> _errorController =
+  StreamController<ErrorAnimationType>();
+
+  // Handles text field
+  TextEditingController _textEditingController = TextEditingController();
+
+  PINEntryController _pinEntryController = Get.find<PINEntryController>();
+
+  Widget buildWidthFillingButton() {
+    return SizedBox(
+      width: double.infinity,
+      child: MojaButton('Set PIN', () {
+        final String enteredPIN = _textEditingController.text;
+        print('Entered PIN: $enteredPIN');
+        if (enteredPIN.length == PINEntryController.PINlength) {
+          _pinEntryController.storeNewPIN(enteredPIN);
+          _textEditingController.clear();
+        }
+      })
+    );
+  }
+
+  Widget buildPINEntryScreen(BuildContext context) {
+    return Container(
+      height: Get.height,
+      width: Get.width,
+      child: Column(
+          mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+          crossAxisAlignment: CrossAxisAlignment.center,
+          children: [
+            Container(width: Get.width - 50,
```

Comment:
Can you fix the indentation here? You may want to check about how the automatic formatting works with trailing commas [here](https://flutter.dev/docs/development/tools/formatting#using-trailing-commas).